src/main/java/screens/Factory.java

```diff
@@ -0,0 +1,37 @@
+package screens;
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * Created by Martijn on 24-5-2017.
+ */
+public class Factory {
```

Comment:
Factory is a rather ambiguous name, could you change the name to give more of an indication what this factory produces?